packages/ui/components/ui/splash-page.tsx

```diff
@@ -1,12 +1,20 @@
 import { ReactNode } from 'react';
-import { Card, CardContent, CardHeader, CardTitle } from './card';
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from './card';
 import { CompressedVideoLogo } from './logo/compressed-video';
 import { FadeTransition } from './fade-transition';
 
-export const SplashPage = ({ title, children }: { title: string; children: ReactNode }) => {
+export const SplashPage = ({
+  title,
+  description,
+  children,
+}: {
+  title: string;
+  description?: ReactNode;
+  children: ReactNode;
+}) => {
   return (
     <FadeTransition>
-      <div className='absolute inset-0 flex w-screen items-center justify-center'>
+      <div className='absolute inset-0 z-[-1] flex w-screen items-center justify-center'>
```

Comment:
Fixes an issue where, for some reason, the background video was floating in front of the page content.